Source/WarpX.cpp

```diff
@@ -508,8 +508,9 @@ WarpX::ReadParameters ()
                 sort_bin_size[i] = vect_sort_bin_size[i];
         }
 
-        double quantum_xi;
-        int quantum_xi_is_specified = pp.query("quantum_xi", quantum_xi);
+        amrex::Real quantum_xi_tmp;
+        int quantum_xi_is_specified = queryWithParser(pp, "quantum_xi", quantum_xi_tmp);
+        double quantum_xi = quantum_xi_tmp;
         if (quantum_xi_is_specified)
             quantum_xi_c2 = quantum_xi * PhysConst::c * PhysConst::c;
```

Comment:
`queryWithParser` does not assign a value to `quantum_xi_tmp` if no value is specified.
That means `double quantum_xi = quantum_xi_tmp;` would assign an undefined value, which we should avoid (even if `wuantum_xi` is then discarded).
```suggestion
        if (quantum_xi_is_specified) {
            double const quantum_xi = quantum_xi_tmp;
            quantum_xi_c2 = quantum_xi * PhysConst::c * PhysConst::c;
        }
```

or
```suggestion
        if (quantum_xi_is_specified)
            quantum_xi_c2 = quantum_xi_tmp * PhysConst::c * PhysConst::c;
```